### PR TITLE
docs: add logo

### PR DESCRIPTION
**Pull request by MrCat33**
**Description**

add logo in docs and readme.md

**Linked Issues**

**Additional context**

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

**Comment by sxzz** [comment]
https://github.com/MrCat33/unplugin-vue-macros/blob/main/README-zh-CN.md 

The size of the image seems to toooo big, we should add `width` property to limit it.

**Review by sxzz — COMMENTED** [review]
https://github.com/sxzz/unplugin-vue-macros/blob/ef313680c1b4d8172316d4247d66aceaab7f276a/docs/.vitepress/config.ts#L16

Please change this too

**Comment by MrCat33** [comment]
done

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img src="./docs/public/logo.svg" style="width:200px;" />
```
Comment (sxzz) [inline README.md:2]:
```suggestion
  <img src="./docs/public/logo.svg" width="200px" />
```

**Pull request merged**